### ISSUE TITLE
Makes the Civicrm admin menu crash on wordpress

### ISSUE THREAD

**Issue by marciojacinto** [osseed/com.osseed.eventcalendar#109]
Makes the admin menu crash on wordpress,
ssystem status dosent works also

**Comment by themak1985** [comment]
Can replicate here. 

Here i the log data from WordPress

[08-Dec-2024 23:52:00 America/New_York] PHP Warning:  Undefined array key "weight" in /home/tarabnyc/www/www/wp-content/plugins/civicrm/civicrm/CRM/Core/Action.php on line 318
[08-Dec-2024 23:52:00 America/New_York] PHP Warning:  Undefined array key "weight" in /home/tarabnyc/www/www/wp-content/plugins/civicrm/civicrm/CRM/Core/Action.php on line 318
[08-Dec-2024 23:52:00 America/New_York] PHP Warning:  Undefined array key "weight" in /home/tarabnyc/www/www/wp-content/plugins/civicrm/civicrm/CRM/Core/Action.php on line 318
[08-Dec-2024 23:52:00 America/New_York] PHP Warning:  Undefined array key "weight" in /home/tarabnyc/www/www/wp-content/plugins/civicrm/civicrm/CRM/Core/Action.php on line 318
[08-Dec-2024 23:52:00 America/New_York] PHP Fatal error:  Smarty error: [in CRM/EventCalendar/Page/ManageEventCalendars.tpl line 29]: syntax error: invalid attribute name: &#039;$rows&#039; (Smarty_Compiler.class.php, line 1549) in /home/tarabnyc/www/www/wp-content/plugins/civicrm/civicrm/packages/Smarty/Smarty.class.php on line 1215

**Comment by themak1985** [comment]
Can confirm - upgrading to code master fixes the issue.

**Comment by sushantpaste** [comment]
@themak1985 Yes. We have not yet released new version you can check master branch for fixes.
